modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,98 @@
-
-You can use the `candidate-{product-version}` channel to update from a previous minor version of {product-title}.
+= Update channels
 
+ifndef::openshift-origin[]
 [id="fast-version-channel_{context}"]
 == fast-{product-version} channel
-
-The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
-
-You can use the `fast-{product-version}` channel to upgrade from a previous minor version of {product-title}.
-
-ifndef::openshift-origin[]
+The `fast-{product-version}` channel is updated with new versions of {product-version} as soon as Red Hat declares the given version as a general availability (GA) release. As such, these releases are fully commercially supported and purposed to be used in production environments.
 
 [id="stable-version-channel_{context}"]
 == stable-{product-version} channel
-While the `fast-{product-version}` channel contains releases as soon as their errata are published, releases are added to the `stable-{product-version}` channel after a delay. During this delay, data is collected from Red Hat SRE teams, Red Hat support services, and pre-production and production environments that participate in connected customer program about the stability of the release.
-
-You can use the `stable-{product-version}` channel to upgrade from a previous minor version of {product-title}.
-endif::openshift-origin[]
-ifdef::openshift-origin[]
+While the `fast-{product-version}` channel contains releases as soon as their errata are published, releases are added to the `stable-{product-version}` channel after a delay. During this delay, data is collected from multiple sources and continuously analyzed for indications of product regressions. Once a statistically significant number of data points have been collected and absent negative signals these releases are added to the stable channel.
 
-[id="stable-4-channel_{context}"]
-== stable-4 channel
-Releases are added to the `stable-4` channel after passing all tests.
+[NOTE]
+Because the time required to obtain a statistically significant number of data points varies based on many factors we offer no Service LeveL Objective (SLO) for the delay duration between fast and stable channels. Please see link:strategies_{context}[Fast and stable channel use and strategies] for more information.
```

Comment:
I used xref to an anchor as it's within the same module, does that work?

Reply:
So we can't use xrefs in module files because it can mess with the software we use to deploy the website. In this case you can say. 

`For more information, please see "Choosing the correct channel for your cluster"`

Then add the xref to the additional resources section in the assembly file (updating/understanding-upgrade-channels-release.adoc)